src/layers/line-layer/line-layer.ts

```diff
@@ -66,20 +68,15 @@ export type LineLayerConfig = Merge<
   {columns: LineLayerColumnsConfig; visConfig: LineLayerVisConfig}
 >;
 
-export const linePosAccessor = ({
-  lat0,
-  lng0,
-  lat1,
-  lng1,
-  alt0,
-  alt1
-}: LineLayerColumnsConfig) => dc => d => [
+export const linePosAccessor = ({lat0, lng0, lat1, lng1, alt0, alt1}: LineLayerColumnsConfig) => (
+  dc: DataContainerInterface
+) => d => [
   dc.valueAt(d.index, lng0.fieldIdx),
   dc.valueAt(d.index, lat0.fieldIdx),
-  alt0?.fieldIdx > -1 ? dc.valueAt(d.index, alt0.fieldIdx) : 0,
+  alt0?.fieldIdx && alt0.fieldIdx > -1 ? dc.valueAt(d.index, alt0.fieldIdx) : 0,
```

Comment:
`alt0?.fieldIdx`  will be falsy when `fieldIdx` is 0, why do you need to check `alt0?.fieldIdx` exists?
if you really need to checkit use `Number.isFinite(alt0?.fieldIdx)` or `notNullOrUndefined(alt0?.fieldIdx)`

Reply:
Please address this comment here

Reply:
I have changed this lines of code already, please check the actual changes:
`alt0 && alt0.fieldIdx > -1 ? dc.valueAt(d.index, alt0.fieldIdx) : 0`